src/crypto.js

```diff
@@ -0,0 +1,175 @@
+'use strict';
+
+const promised = require('./promised.js');
+const green = '\x1b[32m';
+const red = '\x1b[31m';
```

Comment:
Colors in `crypto` module, really?
![Ubp37r9](https://user-images.githubusercontent.com/4405297/120111514-d776f380-c17a-11eb-9faf-0bc49d32bbf9.gif)